assets/stylesheets/components/blog-teaser.scss

```diff
@@ -0,0 +1,19 @@
+.blog .content {
```

Comment:
This is creating a lot of nesting. It would be cleaner and more robust to just call each selector individually. 

maybe rename `.content` to `blog-content` so it isn't such a generic class